lib/ashleighs_rubbish_inject.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+  def inject_method(arg)
+  	array_length = self.length
+  	self[0..array_length].each do |n|
+  	  arg = yield(arg,n)
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.